features/lorawan/lorastack/phy/LoRaPHY.h

```diff
@@ -44,102 +44,163 @@ class LoRaPHY {
     LoRaPHY(LoRaWANTimeHandler &lora_time);
```

Comment:
Move LoRaPHY's constructor as protected. Only inherited classes can construct this class. Also default and copy constructors should be prevented.